test/forms/behaviours/FieldBehaviours.scala

```diff
@@ -45,6 +45,16 @@ trait FieldBehaviours extends FormSpec with ScalaCheckPropertyChecks with Genera
       }
     }
 
+  def fieldWithMinLengthAlpha(form: Form[_], fieldName: String, maxLength: Int, lengthError: FormError): Unit =
```

Comment:
I'm not sure the logic is right here. The behaviour is `fieldWithMinLengthAlpha` which suggests the parameter for it should be `minLength` and it should generate strings longer than that min length?

Reply:
See most recent commit below. The generator used in this method generates a number between 1 and 18 ("minLength -1") and checks to see if that number == setLength if not it will invoke the length error